src/parsers/pm2ps_parser.jl

```diff
@@ -79,7 +79,10 @@ function make_load(d, bus)
     return PowerLoad(;
         name=bus.name,
         available=true,
+        model = PowerSystems.ConstantPower::LoadModel,
```

Comment:
You should be able to use ConstantPower::LoadModel since you are within PowerSystems.

Reply:
It seems that all over the JSON file internals are always called using `PowerSystems.`

Reply:
This is inside our parsing code.  There is no need for prefixing with PowerSystems.

Reply:
I still prefer to leave them for reference to where they come from in the JSON.

Reply:
PowerSystems.ConstantPower::LoadModel doesn't come from JSON.  You defined it in common.jl.

Reply:
I meant to say such that one can now if reading the JSON file where does that come from.